pkg/registry/service/registry_test.go

```diff
@@ -1,335 +0,0 @@
-/*
```

Comment:
FYI: I'm removing this test, because it's now fully covered by pkg/registry/service/etcd/etcd_test.